vagrant-centos-kubeadm-v1.9.2/00-windows.ps1

```diff
@@ -0,0 +1,43 @@
+Param(
+  [String]$pauseimagename
+)
+
+docker import \baseimages\CBaseOs_rs_onecore_stack_17089.1000.180129-1709_amd64fre_NanoServer_en-us.tar.gz microsoft/nanoserver-insider
```

Comment:
Where do these base images come from?

Reply:
These are internal builds. I'll update to use the latest public, Insider builds working towards Windows Server version 1803 (RS4)